client/v2/client.go

```diff
@@ -19,6 +19,7 @@ import (
 // could be travelling over the internet.
 const (
 	UDPPayloadSize = 512
+	MaxPointSize   = 64 * 1024
```

Comment:
Assuming IPV4, a safer payload size limit would be 65467 = 65535 (2^16-1) - 8 (UDP Header bytes) - 60 (Largest possible IPv4 Header bytes). 65507 could be used if no IPv4 options are used.